### ISSUE TITLE
Studies normalizeren.

### ISSUE THREAD

**Issue by qurben** [csrdelft/csrdelft.nl#364]
Het studie veld in profiel is een zooi.

Voor WB is er  'Werktuigbouwkunde', 'werktuigbouwkunde' en 'TU Delft - WB'.

Het zou fijn zijn als er wat meer eisen worden gesteld aan dit veld, zodat mensen niet zomaar hun eigen studie kunnen verzinnen.

**Comment by Erackron** [comment]
Echt restricties op dit veld gooien lijkt me niet gewenst, maar misschien wel laten zoeken en eventueel een nieuwe studie toevoegen als dat nodig is zou het al wel verhelpen denk ik.

**Comment by qurben** [comment]
Ja, precies. Een manier verzinnen waardoor het opgeruimd blijft.
